.github/workflows/rust.yml

```diff
@@ -238,23 +238,17 @@ jobs:
     - uses: hecrj/setup-rust-action@v1
       with:
         rust-version: stable
-        targets: x86_64-unknown-linux-musl
+        targets: x86_64-unknown-linux-gnu
     - uses: Swatinem/rust-cache@v2
       with:
         key: static-linux-binary
-    - name: Install musl tools
-      run: |
-        sudo apt-get update
-        sudo apt-get install -y --no-install-recommends musl-tools
     - uses: actions/checkout@v3
     - name: Build release binary
-      run: cargo build --release --target x86_64-unknown-linux-musl
-    - name: Strip binary from debug symbols
-      run: strip target/x86_64-unknown-linux-musl/release/fnm
```

Comment:
no need for strip anymore [since Cargo 1.59](https://blog.rust-lang.org/2022/02/24/Rust-1.59.0.html#creating-stripped-binaries)